src/components/ReminderCtaButton.tsx

```diff
@@ -134,28 +134,20 @@ export const ReminderCtaButton = ({
         case 'DEFAULT':
             return (
                 <div css={styles.buttonWrapperStyles}>
-                    <RemindMeButton
-                        buttonStyles={styles}
-                        onClick={onClick}
-                        disabled={false}
-                        ctaText={reminderCta}
-                    />
-                    <div css={styles.smallPrint}>
-                        We will send you a maximum of two emails in {reminderLabel}. To find out
-                        what personal data we collect and how we use it, view our{' '}
-                        <a href="https://manage.theguardian.com/email-prefs">Privacy Policy</a>.
-                    </div>
+                    <RemindMeButton buttonStyles={styles} onClick={onClick} ctaText={reminderCta} />
+                    {!isBanner && (
```

Comment:
Oh - do we not want to show this text on the banner? Is it a data privacy requirement to show it?

If it's okay to hide it on the banner, I'd be in favour of controlling it more explicitly i.e. with a prop called `hidePrivacyText` or similar. That feels more intuitive and robust to me than coupling to possible consumers in here. What do you think?

Reply:
This is a controversial change I've introduced. The Epic includes the message because that's the message that shows up after user clicks on reminder button in RRCP banner/epics - but in BrazeLand the reminder button is always going to be a single-click action. I cleared adding the copy with Marketing in chat when doing the work to update the Epic reminder to the RRCP way-of-things. But when adding the reminder button to the Banner we run into the vertical space issue - the copy under the reminder button can take 2-4 lines of additional vertical space!

`hidePrivacyText` sounds like a good solution for the copy on both epic and banner! It does add an extra attribute which Marketing will need to be aware of when building epic/banner messages in Braze canvases - another thing that can be fixed through education?

I'll give it a go.

Reply:
Oh sorry that's actually not quite what I meant but is definitely an alternative & a valid approach! I just meant rename the current prop in this specific component to `shouldHidePrivacyText` or similar. Allowing marketing to configure as you suggested would definitely give the most flexibility. Perhaps worth asking around first in case there's a data privacy requirement to _always_ show this text? I'm just thinking that if we should always be showing it then it wouldn't be worth making it configurable.

Reply:
(But yes I also totally see your point about spacing on the banner!)

Reply:
Done

Reply:
So now we're making the privacy text optional and controllable from Braze for both banners and epics? If that's okay data-privacy wise then that sounds good 👍 I'd be interested in which the usual case is (to hide it or to show it), if we should usually show it I wonder if that should be the default? I.e. should the prop be `shouldHidePrivacyText` and that defaults to `false` (i.e. show the text) if omitted in the key value pairs.

Reply:
In the context of a Braze banner/epic, where the reader has already signed in and thus (in theory) already knows we have their email address. I'm not convinced we need to show the privacy notice in all cases? I assume Braze itself can handle winnowing out users who ask for a reminder but have also set their communication preference to not receiving email. But (as always) I'm not a data privacy expert ...

I've set the default to `false` for epics. I've set it to `true` for banners, mainly because of the need to minimise the banner's vertical size.